sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/utils/CxxReportSensor.java

```diff
@@ -177,13 +177,16 @@ protected String getStringProperty(String name, String def) {
       directoryScanner.setIncludes(includes.toArray(new String[includes.size()]));
       directoryScanner.scan();
 
-      for (String found : directoryScanner.getIncludedFiles()) {
-        CxxUtils.LOG.debug("Adding report '{}'", found);
+      String [] includeFiles = directoryScanner.getIncludedFiles();
```

Comment:
looks like this now:

[00:51:30] :		 [Step 3/3] 00:51:30.187 INFO  - Sensor CxxRatsSensor
[00:51:30] :		 [Step 3/3] 00:51:30.195 INFO  - Scanner found '227' report files
[00:51:30] :		 [Step 3/3] 00:51:30.195 INFO  - Parser will parse '227' report files
[00:51:30] :		 [Step 3/3] 00:51:30.195 INFO  - Processing report 'E:\prod\structures\src\work\Core\Reporting\.cxxresults\reports-rats\rats-result-Reporting-0.xml'
[00:51:30] :		 [Step 3/3] 00:51:30.196 INFO  - Processing report 'E:\prod\structures\src\work\Core\Reporting\.cxxresults\reports-rats\rats-result-Reporting-1.xml'

and in debug mode shows the number of issues, i think its good